.eslintrc

```diff
@@ -1,46 +0,0 @@
-{
-  "env": {
-    "es6": true,
-    "browser": true,
-    "jasmine": true,
-    "protractor": true,
-    "node": true,
-    "jest": true
-  },
-  "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "comment": true,
-    "ecmaFeatures": {
-      "jsx": true
-    },
-    "ecmaVersion": 2017,
-    "sourceType": "module"
-  },
-  "plugins": ["prettier", "react", "react-hooks", "@typescript-eslint"],
-  "extends": [
-    "eslint:recommended",
-    "plugin:cypress/recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:react/recommended",
-    "plugin:prettier/recommended",
-    "prettier/@typescript-eslint",
-    "prettier/react"
```

Comment:
IIUC these are from eslint-config-prettier and ensure that there are no collisions between eslint and prettier rules. Shouldn't we keep these?

https://prettier.io/docs/en/integrating-with-linters.html